.ibm/pipelines/utils.sh

```diff
@@ -744,9 +764,24 @@ install_acm_operator(){
     [[ "$CURRENT_PHASE" == "Running" ]] && echo "MulticlusterHub is now in Running phase." && break
     sleep 10
   done' || echo "Timed out after 15 minutes"
-
 }
 
+# TODO
```

Comment:
@zdrapela Do we need this commented out? :)

Reply:
No, thanks :-) I deal with it differently now.